README.md

```diff
@@ -58,6 +58,15 @@ If you use Rails in the default mode without any pre-processor, you'll have to a
 
 If you followed the [official bootstrap installation guide](https://github.com/twbs/bootstrap-rubygem#a-ruby-on-rails), you'll probably have switched to SCSS. In this case add the following line to your `application.scss`:
 
+
```

Comment:
I don't have enough GitHub fu to know how to do this better, so what I would suggest is something like move your added lines to after line 52. The current line 53 would become the last paragraph. We'd also have to update the wording, because it's been a long time since that was likely the "default" way of adding.

If you make those changes, then maybe I can make any other small edits via GitHub suggestions.